test/e2e/run.sh

```diff
@@ -1059,6 +1061,7 @@ cri_resmgr_extra_args=${cri_resmgr_extra_args:-""}
 cri_resmgr_agent_extra_args=${cri_resmgr_agent_extra_args:-""}
 cleanup=${cleanup:-0}
 reinstall_all=${reinstall_all:-0}
+reinstall_bootstrap=${reinstall_bootstrap:-0}
```

Comment:
nitpick: I guess this is a(n orthogonal) fixup for the origin commit introducing the `reinstall_bootstrap` flag.

Reply:
Yeah, your guess is right.

Our shellcheck git hook bit me when I was trying to create a commit that modifies run.sh.  Fixing the shellcheck issue here was the least time consuming but the second ugliest choice. It definitely would have been nicer to disable the shellcheck hook for some time and create two PRs. And it would have been even more evil to leave this unfixed and let it bite someone again in the future. So this was a non-optimal compromise that won't make anyone very happy...

Encouraged by your comment of "no need to split", I could leave this as is and feel the sting I deserved. :)